prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/AR/AR-receiver-transform.yml

```diff
@@ -0,0 +1,223 @@
+        Chicot: 05017
+        Clark: 05019
+        Clay: 05021
+        Cleburne: 05023
+        Cleveland: 05025
+        Columbia: 05027
+        Conway: 05029
+        Craighead: 05031
+        Crawford: 05033
+        Crittenden: 05035
+        Cross: 05037
+        Dallas: 05039
+        Desha: 05041
+        Drew: 05043
+        Faulkner: 05045
+        Franklin: 05047
+        Fulton: 05049
+        Garland: 05051
+        Grant: 05053
+        Greene: 05055
+        Hempstead: 05057
+        Hot Spring: 05059
+        Howard: 05061
+        Independence: 05063
+        Izard: 05065
+        Jackson: 05067
+        Jefferson: 05069
+        Johnson: 05071
+        Lafayette: 05073
+        Lawrence: 05075
+        Lee: 05077
+        Lincoln: 05079
+        Little River: 05081
+        Logan: 05083
+        Lonoke: 05085
+        Madison: 05087
+        Marion: 05089
+        Miller: 05091
+        Mississippi: 05093
+        Monroe: 05095
+        Montgomery: 05097
+        Nevada: 05099
+        Newton: 05101
+        Ouachita: 05103
+        Perry: 05105
+        Phillips: 05107
+        Pike: 05109
+        Poinsett: 05111
+        Polk: 05113
+        Pope: 05115
+        Prairie: 05117
+        Pulaski: 05119
+        Randolph: 05121
+        St. Francis: 05123
+        Saline: 05125
+        Scott: 05127
+        Searcy: 05129
+        Sebastian: 05131
+        Sevier: 05133
+        Sharp: 05135
+        Stone: 05137
+        Union: 05139
+        Van Buren: 05141
+        Washington: 05143
+        White: 05145
+        Woodruff: 05147
+        Yell: 05149
+    hl7Spec: [ '%{PID}-11-9' ]
+
+  # converts value "PRS" to "PRN" for AR who uses table HL70201 from Ver 2.5.1. RS default is Ver 2.7
+  - name: ar-patient-telecommunication-use-code
+    resource: 'Bundle.entry.resource.ofType(Patient).telecom.extension("https://reportstream.cdc.gov/fhir/StructureDefinition/xtn-contact-point").extension("XTN.2")'
+    condition: '%resource.value = "PRS"'
+    value: [ "'PRN'"]
+    hl7Spec: [ '%{PID}-13-2' ]
+
+    # add "NPI" to ORC-12-9-1, ORC-12-9-2, OBR-16-9-1 and OBR-16-9-2
+  - name: ar-ordering-provider-assigning-authority-id
+    condition: 'Bundle.entry.resource.ofType(Practitioner).identifier.system = "http://hl7.org/fhir/sid/us-npi"'
+    value: [ '"NPI"']
+    hl7Spec: [ '%{ORC}-12-9-1', '%{ORC}-12-9-2' , '%{OBR}-16-9-1' , '%{OBR}-16-9-2' ]
+
+    # add "ISO" to ORC-12-9-3 and OBR-16-9-3
+  - name: ar-ordering-provider-assigning-authority-namespace-universal-id-type
+    condition: 'Bundle.entry.resource.ofType(Practitioner).identifier.system = "http://hl7.org/fhir/sid/us-npi"'
+    value: [ '"ISO"' ]
+    hl7Spec: [ '%{ORC}-12-9-3' , '%{OBR}-16-9-3'  ]
+
+    # add "NPI" to ORC-12-13 and OBR-16-13
+  - name: ar-ordering-provider
+    condition: 'Bundle.entry.resource.ofType(Practitioner).identifier.system = "http://hl7.org/fhir/sid/us-npi"'
+    value: [ '"NPI"' ]
+    hl7Spec: [ '%{ORC}-12-13-1' , '%{OBR}-16-13-1' ]
+
```

Comment:
Check if you still need to add the NPI transform after Jessica's changes to the simple report sender transform.